index.html

```diff
@@ -0,0 +1,126 @@
+        <p class="black location">@National Museum of Korea, Art Center Nabi and more</p>
+    </section>
+    <!--THE MAIN PROGRAM SECTION-->
+    <section class="second-section">
+        <h2 class="white">Main Program</h2>
+        <div class="second-section-container">
+            <div class="prog-feature">
+                <img src="./assets/icon-1.png" alt="Lecturer">
+                <h4 class="orange">Lecture</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-2.png" alt="CC Exhibition">
+                <h4 class="orange">Exhibition</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-3.png" alt="Forum">
+                <h4 class="orange">Forum</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-4.png" alt="Workshop">
+                <h4 class="orange">Workshop</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+            <div class="prog-feature">
+                <img src="./assets/icon-5.png" alt="CC Ignite">
+                <h4 class="orange">CC Ignite</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
+        </div>
+        <a href="#" class="white logo-link-1">Join CC Global Summit 2015</a>
+        <a href="#" class=" logo-link-2 white">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <!--THE FEATURED SPEAKERS SECTION-->
+    <section class="third-section">
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
+    </section>
+    <footer>
+        <div class="partners">
+            <p class="grey partners-heading">Partners</p>
+            <ul class="partners-list">
+                <li class="grey">mozilla</li>
+                <li class="grey">Google</li>
+                <li class="grey">NAVER</li>
+                <li class="grey">daumkakao</li>
+                <li class="grey">airbnb</li>
+            </ul>
+        </div>
+        <div class="footer-signoff">
+            <div class="footer-logo">
+                <i class="fa fa-life-ring fa-3x orange"></i>
+                <p>creative commons korea</p>
+            </div>
+            <div class="footer-logo-text">
+                <p>2015 Creative Commons Korea. Some Rights Reserved</p>
+                <p>This project is a personal inspiration to showcase my skills and all content therein is not actual reality.</p>
```

Comment:
- [ ] The font style is not correct for this footer section. As per project requirements, we have to use cocogoose font-family. Let's try to add some more padding so that it can match the original design. Kindly try to fix this issue.

Your 
![image](https://user-images.githubusercontent.com/53019916/127885082-71df6658-4a93-4344-9d9a-778d64501206.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127885164-b30bc242-cebe-4a9d-ac45-3cd3681876a1.png)